src/utils/dump.sh

```diff
@@ -8,7 +8,7 @@
 cmd=run.pl
 do_delta=false
 nj=32
-
+verbose=1
```

Comment:
You can use `verbose=0` as default, and remove `--verbose ${verbose}` in `run.sh`